packages/iov-keycontrol/src/userprofile.spec.ts

```diff
@@ -102,46 +102,51 @@ describe("UserProfile", () => {
 
   it("can add entries", () => {
     const profile = new UserProfile();
+    let entry1;
+    let entry2;
+    let entry3;
     expect(profile.wallets.value.length).toEqual(0);
     expect(profile.wallets.value.map(i => i.label)).toEqual([]);
-    profile.addEntry(Ed25519HdWallet.fromMnemonic("melt wisdom mesh wash item catalog talk enjoy gaze hat brush wash"));
+    profile.addEntry((entry1 = Ed25519HdWallet.fromMnemonic("melt wisdom mesh wash item catalog talk enjoy gaze hat brush wash")));
```

Comment:
I am not so happy with the combined assignment/return. Better pre-create the 3 entries and make them const instead of let